src/Makefile

```diff
@@ -29,9 +29,12 @@ CFLAGS_AUTO = -g -pipe -fno-unwind-tables -fno-asynchronous-unwind-tables -ffunc
 -Werror=implicit-function-declaration -Werror=implicit-int -Werror=pointer-sign -fPIC
 CFLAGS_C99FSE = -std=c99 -ffreestanding -fexcess-precision=standard -frounding-math -Wa,--noexecstack
 
+LIBVICSETUP_DIR = $(srcdir)/vicsetup/libvicsetup
+ARGON2_DIR = $(SGXLKL_ROOT)/third_party/argon2
+
 CFLAGS_SGXLKL_ALL = $(CFLAGS_C99FSE) $(CFLAGS_AUTO) $(CFLAGS_MAIN) $(MUSL_CFLAGS)
 CFLAGS_SGXLKL_ALL += -D_XOPEN_SOURCE=700
-CFLAGS_SGXLKL_ALL += $(SGXLKL_INCLUDES) -isystem $(CRYPTSETUP_BUILD)/include -isystem $(E2FSPROGS_BUILD)/include -I$(srcdir)/libjson/include
+CFLAGS_SGXLKL_ALL += $(SGXLKL_INCLUDES) -isystem $(E2FSPROGS_BUILD)/include -I$(srcdir)/libjson/include -I$(LIBVICSETUP_DIR)/include -I$(ARGON2_DIR)/include
```

Comment:
Argon2 should be using `-isystem` since it's not our code.

Reply:
Fixed. The gcc man pages is vague on what the different is.